dali/base/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ set (    INCLUDES
 include_directories ( 
          ./../../common/deftype 
          ./../../common/remote 
+         ./../../common/workunit
```

Comment:
NO! Dali should not be dependent on workunit.

Reply:
Dali does not link with workunit, it only includes that file so it can resolve isWorkunitDAToken()

Reply:
It still shows an incorrect dependency.